### DISCUSSION TITLE
Nickname / Volume name has -1 suffix

### DISCUSSION THREAD

**Discussion by BrainPonders** [iterate-ch/cyberduck#15049] [Q&A] [answered]
During connection issues I reconnected a bookmark, making it appear twice. The second instant has a -1 suffix to the original name in the finder favourites.

No matter what I try, I cannot get rite of the -1 suffix. Including deleting all bookmarks and recreate them. This particular bookmark always comes back with -1 suffix in finder.

The bookmark configuration file has the correct name.

**To Reproduce**
Unable to describe as the situation is the result after a unknown connection issue.

**Expected behavior**
To be able to rename the name in finder.

**Desktop (please complete the following information):**
 - OS: MacOS
 - Version 13.5.1

**Mountain Duck**
Version 4.14.3 (21432)

**Comment by tatze96** [comment, marked as answer]
There might be an abandoned mount item left within the _Sessions_ folder that causes the -1 suffix on the bookmark name.

Please delete the said folder from within the application support folder to clear out any active sessions and try again to mount the bookmark. Ensure that both Mountain Duck and Cyberduck aren't running while doing so.

Refer to our [wiki page](https://docs.cyberduck.io/mountainduck/support/#application-support-folder) for more information on the location of the application support folder.

**Reply by BrainPonders** [reply]
I had to reboot the computer in order to make this work.
